Calendula/src/test/java/es/usc/citius/servando/calendula/database/DBTest.kt

```diff
@@ -93,4 +95,18 @@ class DBTest {
         assertEquals(DB.medicines().findAll()[0].name, MED_NAME)
         assertEquals(DB.schedules().findAll()[0].items().size.toLong(), 1)
     }
+
+    /**
+     * Abuse test framework to export database into a project folder, allowing further analysis of database schema
+     */
+    @Test
+    fun exportDatabase() {
+        val db = DB.helper().writableDatabase;
+        DB.helper().close();
+        val destination = File("../db")
```

Comment:
```suggestion
        val destination = File("../generated-db")
```

`db` is probably too generic, let's change it to generated-db or something.